[2025-04-11T07:52:34.359+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: proyecto_bigdata.guardar_postgresql manual__2025-04-11T07:52:28.287106+00:00 [queued]>
[2025-04-11T07:52:34.367+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: proyecto_bigdata.guardar_postgresql manual__2025-04-11T07:52:28.287106+00:00 [queued]>
[2025-04-11T07:52:34.368+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-11T07:52:34.381+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): guardar_postgresql> on 2025-04-11 07:52:28.287106+00:00
[2025-04-11T07:52:34.387+0000] {standard_task_runner.py:60} INFO - Started process 370 to run task
[2025-04-11T07:52:34.390+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'proyecto_bigdata', 'guardar_postgresql', 'manual__2025-04-11T07:52:28.287106+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/proyecto_bigdata_dag.py', '--cfg-path', '/tmp/tmpyjo2wbxd']
[2025-04-11T07:52:34.392+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask guardar_postgresql
[2025-04-11T07:52:34.436+0000] {task_command.py:423} INFO - Running <TaskInstance: proyecto_bigdata.guardar_postgresql manual__2025-04-11T07:52:28.287106+00:00 [running]> on host 04c54140d555
[2025-04-11T07:52:34.513+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='proyecto_bigdata' AIRFLOW_CTX_TASK_ID='guardar_postgresql' AIRFLOW_CTX_EXECUTION_DATE='2025-04-11T07:52:28.287106+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-11T07:52:28.287106+00:00'
[2025-04-11T07:52:34.573+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-11T07:52:34.584+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=proyecto_bigdata, task_id=guardar_postgresql, execution_date=20250411T075228, start_date=20250411T075234, end_date=20250411T075234
[2025-04-11T07:52:34.604+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-11T07:52:34.628+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
